instrumentation/reactor-3.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/reactor/ReactorHooksAdvice.java

```diff
@@ -5,12 +5,15 @@
 
 package io.opentelemetry.javaagent.instrumentation.reactor;
 
+import io.opentelemetry.instrumentation.api.tracer.async.AsyncSpanEndStrategies;
+import io.opentelemetry.instrumentation.reactor.ReactorAsyncSpanEndStrategy;
 import io.opentelemetry.instrumentation.reactor.TracingOperator;
 import net.bytebuddy.asm.Advice;
 
 public class ReactorHooksAdvice {
   @Advice.OnMethodExit(suppress = Throwable.class)
   public static void postStaticInitializer() {
     TracingOperator.registerOnEachOperator();
+    AsyncSpanEndStrategies.getInstance().registerStrategy(ReactorAsyncSpanEndStrategy.INSTANCE);
```

Comment:
How about adding this to `TracingOperator`? In the RxJava2 instrumentation the async span strategy is also usable by the library instrumentation.